activities/tool-management/index.md

```diff
@@ -4,20 +4,23 @@ type: index
 
 # Tool management and guidance
 
-The Foundation for Public Code needs tools to accomplish its mission. This is how we manage our tools:
+The Foundation for Public Code needs tools to accomplish its mission. We use a variety of digital tools. Our tools range from being entirely self-managed, having service providers, and completely externally managed. The Operations Coordinator is the system administrator of all the self-managed tools. They can create accounts and ensure updates happen regularly. The Chief Executive can do this as well. Most of these tools are used for our internal processes, and some are available to our community.
+
+## This is how we manage our tools
 
 * [Tool and email naming conventions](tool-and-email-naming-conventions.md)
 * [How to test a tool](tool-testing.md)
 
 ## Tool guidance
 
-This guidance covers:
+Our guidance covers:
 
-* instructions for users
```

Comment:
List items that follow on from a lead-in sentence (where the bullets replace a comma-separated list) should be lowercase.